packages/file-format-ts/src/index.ts

```diff
@@ -0,0 +1,5 @@
+// Types must be generated using using `yarn generate` before they can
```

Comment:
Furthermore, the schemas must be built too. It might be worth simply stating that `yarn build` should be run in the monorepo root in order to make this file start to work ...

Reply:
Good point, I updated the comment with more context.